spec/controllers/admin/event_schedules_controller_spec.rb

```diff
@@ -0,0 +1,143 @@
+require 'spec_helper'
+
+describe Admin::EventSchedulesController do
+  let(:venue) { create(:venue) }
+  let(:conference) { create(:conference, venue: venue) }
+  let(:schedule) { create(:schedule, program: conference.program)}
+  let(:event_schedule) { create(:event_schedule, schedule: schedule)}
+  let!(:organizer_role) { Role.find_by(name: 'organizer', resource: conference) }
+  let(:organizer) { create(:user, role_ids: organizer_role.id) }
+
+  context 'logged in as an organizer' do
+    before :each do
+      sign_in(organizer)
+      event_schedule
+    end
+
+    describe 'POST #create' do
+      context 'with valid attributes' do
+        it 'saves the event schedule to the database' do
+          expected = expect do
+            post :create, conference_id: conference.short_title, event_schedule:
+                 attributes_for(:event_schedule,
+                                schedule_id: schedule.id,
+                                event_id: create(:event, program: conference.program).id,
+                                room_id: create(:room, venue: venue).id,
+                                start_time: conference.start_date)
+          end
+          expected.to change { EventSchedule.count }.by 1
```

Comment:
Because Rubocop complains:

> Avoid using {...} for multi-line blocks in Rspec change block

There are already other tests doing the same, for example https://github.com/openSUSE/osem/blob/master/spec/controllers/admin/conferences_controller_spec.rb#L81

Some others don't define the expected variable, like this one: https://github.com/openSUSE/osem/blob/master/spec/controllers/admin/sponsorship_levels_controller_spec.rb#L64. But I don't like writing `end.to change`